v2/internal/frontend/desktop/linux/gtk.go

```diff
@@ -0,0 +1,73 @@
+package linux
+
+/*
+#cgo linux pkg-config: gtk+-3.0 webkit2gtk-4.0
+
+#include "gtk/gtk.h"
+
+static GtkCheckMenuItem *toGtkCheckMenuItem(void *pointer) { return (GTK_CHECK_MENU_ITEM(pointer)); }
+
+extern void blockClick(GtkWidget* menuItem, gulong handler_id);
+extern void unblockClick(GtkWidget* menuItem, gulong handler_id);
+*/
+import "C"
+import "unsafe"
+import "github.com/wailsapp/wails/v2/pkg/menu"
+
+func GtkMenuItemWithLabel(label string) *C.GtkWidget {
+	cLabel := C.CString(label)
+	result := C.gtk_menu_item_new_with_label(cLabel)
+	C.free(unsafe.Pointer(cLabel))
```

Comment:
Do we need to worry about a similar race condition like we saw before in e713c439f0656b57bd77a06b27aa839058f5e99e?

Reply:
Nah, this takes a copy immediately and stays in scope